phone_verify/services.py

```diff
@@ -25,6 +25,9 @@ class PhoneVerificationService(object):
     except AttributeError:
         raise ImproperlyConfigured("Please define PHONE_VERIFICATION in settings")
 
+    # TODO: Check if the settings are properly configured.
+    # Specially for breaking changes such as addition of `VERIFY_SECURITY_CODE_ONLY_ONCE`
```

Comment:
Can you merge the latest master in your branch, please?

Reply:
Yeah sure!